ydb/core/tx/schemeshard/ut_helpers/export_reboots_common.cpp

```diff
@@ -11,20 +11,36 @@ using namespace NKikimrSchemeOp;
 namespace NSchemeShardUT_Private {
 namespace NExportReboots {
 
+using TCreateHandler = std::function<void(TTestActorRuntime&, ui64&, const TString&, const TString&)>;
+
+#define GEN_CREATE_HANDLER(func) \
+    TCreateHandler([](TTestActorRuntime& runtime, ui64& txId, const TString& parentPath, const TString& scheme) { \
+        func(runtime, txId, parentPath, scheme); \
+    })
+
+THashMap<NKikimrSchemeOp::EPathType, TCreateHandler> CreateHandlers = {
+    {EPathTypeTable, GEN_CREATE_HANDLER(TestCreateTable)},
```

Comment:
Почему не просто:
```suggestion
    {EPathTypeTable, &TestCreateTable},
```
?

Reply:
у `TestCreateTable` и `TestCreateView` и `TestCreateCdcStream` различные сигнатуры, но у них совпадают обязательные аргументы, соответсвенно, типы ссылок разные, поэтому сделал обертку, вызывающую функцию только с обязательными аргументами. Можно бы было сделать не через макрос, а через шаблон, но эти функции перегружены (компилятор вывести тип не может, а указывать его руками неприятно)

Reply:
Тогда сделай над TestSmth обертку-функтор, которая запомнит у себя указатель на функцию и вызовет её с нужными тебе аргументами.